community-membership.md

```diff
@@ -0,0 +1,78 @@
+# Community Membership
+
+## Maintainer
+
+Maintainers are the planners and designers of the project, with the authority to merge branches into the master. Maintainers come from Committers. They must demonstrate good judgment and responsibility for the health of the subproject. The maintainer must set up the technical direction directly or by delegating these responsibilities, and make or approve design decisions for the subproject.
+
+### How to become a Maintainer:
+
+- Demonstrate a deep understanding of TiKV's technical goals and directions
+- Actively organize technical seminars and propose constructive projects
```

Comment:
This sounds more like a responsibility...

Reply:
The Maintainer need to show their ability first, by deeply involved in TiKV and propose constructive projects

Reply:
Then maybe  "Actively involved in organizing technical seminars and proposing constructive subprojects"?

Reply:
done